vendor/vendor.json

```diff
@@ -1,12 +1,12 @@
 {
 	"comment": "",
+	"heroku": {
+		"goVersion": "go1.9",
+		"install": [
+			"./cmd/..."
```

Comment:
@amerine This is passing all dirs/subdirs under cmd as the value for "install"? What happens with this value?

Reply:
I’m not familiar enough with how the platform handles Go in this case. But `./...` is the same as `./*.go ./**/*.go` under the Go tool chain, so presumably that’s how you tell it to use everything in that directory as the primary application.

Reply:
Just realized that wasn’t directed at me. 🤷🏻‍♂️

Reply:
Looks like pre 1.9 this would find all packages/subpackages under vendor. Now it does everything under cmd 

https://github.com/heroku/heroku-buildpack-go#dep-specifics

So any dir with a main file under cmd gets compiled and made available as an executable?

@amerine This means as far as deps go those get explicitly imported in the code?

Reply:
Again, you two have it perfect! 👍